charts/incubator/lldap/values.yaml

```diff
@@ -0,0 +1,55 @@
+image:
+  repository: nitnelave/lldap
+  pullPolicy: IfNotPresent
+  tag: latest
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 17170
+        port: 17170
+  ldap:
+    ports:
+      main:
+        targetPort: 3890
+        port: 3890
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            GID: "0"
+            LLDAP_LDAP_BASE_DN: "dc=example,dc=com"
+            LLDAP_LDAP_USER_PASS: "change me"
+            LLDAP_JWT_SECRET:
+              - secretRef:
+                  name: secrets
+                  key: LLDAP_JWT_SECRET
+            UID: "0"
+# Set lldap variables
+lldap:
+  configuration:
+    ## JWT secret set to auto generate if not defined.
+    # jwt:
+
+persistence:
+  data:
+    enabled: true
+    mountPath: "/data"
```

Comment:
Currently the fsGroup, user and group is set to 0. The readOnlyRootFilesystem and runAsNonRoot is set to false. This fixed the chown with the root file system. However for this mounted volume I'm still getting "chown: /data/lldap_config.toml: Operation not permitted". I'm interpreting this as a permissions thing but can't see what else I can do to fix this.

Reply:
We drop all capabilities, that prevents the chown.

that being said, I personally consider hard-failing chown in containers a bug.